src/Lightning.ts

```diff
@@ -174,23 +174,20 @@ export const LightningMixin = (superclass) => class extends superclass {
         total_mtokens: payment ? mtokens : undefined,
       }));
     } catch (err) {
-      const error = "error getting route / probing for route"
-      lightningLogger.error({ err, max_fee, probingSuccess: false, success: false }, error)
-      throw new LoggedError(error)
+      const error = "Error probing for route"
```

Comment:
I think there is no need to create an error message here, if this will be always the same across all the `RouteFindingError`. you can hard code the message directly within `RouteFindingError` so that you don't have to pass an error message independantly. (this apply to a lot of other error as well that are using the same error message across all their instances)

Reply:
similarly for the error just above this one, `forwardToClient: false` is probably always the same across all the instance of an error